planner-api/src/main/java/pl/edu/agh/kuce/planner/balance/persistence/BalanceRepository.java

```diff
@@ -0,0 +1,12 @@
+package pl.edu.agh.kuce.planner.balance.persistence;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Collection;
+
+public interface BalanceRepository extends JpaRepository<Balance, Integer> {
+
+    @Query("SELECT b FROM Balance b WHERE b.userId = ?1")
```

Comment:
After changing the PR according to https://github.com/marszub/savings-planner/tree/master/planner-api/src/main/java/pl/edu/agh/kuce/planner/event/persistence, you can delete @Query